vagrant/2/1/README.md

```diff
@@ -0,0 +1,6 @@
+Use:
+ - "vagrant status" to see the running machines.
+ - "vagrant ssh" to ssh the machine. 
+
+In case there are more then one machine use:
```

Comment:
This section is not needed for this challenge. The Vagrantfile has only one machine configured.